datumaro/plugins/data_formats/segment_anything/exporter.py

```diff
@@ -69,85 +161,27 @@ def apply(self):
-                            if annotation.type == AnnotationType.bbox:
-                                replace(annotation_data, "bbox", annotation.get_bbox())
-                            elif annotation.type == AnnotationType.mask:
-                                if hasattr(annotation, "rle"):
-                                    rle_encoded = annotation.rle
-                                else:
-                                    rle_encoded = mask_utils.encode(annotation.image)
-                                if isinstance(rle_encoded["counts"], bytes):
-                                    rle_encoded["counts"] = rle_encoded["counts"].decode()
-                                replace(annotation_data, "segmentation", rle_encoded)
-                                replace(annotation_data, "area", annotation.get_area())
-                            else:
-                                continue
-
-                            replace(
-                                annotation_data,
-                                "predicted_iou",
-                                annotation.attributes.get("predicted_iou", 0.0),
-                            )
-                            replace(
-                                annotation_data,
-                                "stability_score",
-                                annotation.attributes.get("stability_score", 0.0),
-                            )
-                            replace(
-                                annotation_data,
-                                "crop_box",
-                                annotation.attributes.get("crop_box", []),
-                            )
-                            replace(
-                                annotation_data,
-                                "point_coords",
-                                annotation.attributes.get("point_coords", [[]]),
-                            )
-
-                        if annotation_data["segmentation"] is NOTSET:
-                            continue
-                        if (
-                            annotation_data["bbox"] is NOTSET
-                            and annotation_data["segmentation"] is not NOTSET
-                        ):
-                            annotation_data["bbox"] = (
-                                annotation_data["segmentation"].get_bbox().tolist()
-                            )
-                        json_data["annotations"].append(annotation_data)
-
-                    if not json_data["annotations"]:
+                    instances = self.find_instances(item.annotations)
+                    annotations = [self.get_annotation_info(i, width, height) for i in instances]
+                    annotations = [i for i in annotations if i is not None]
+                    if not annotations:
+                        log.warning(
+                            "Item '%s': skipping writing instances since no annotation available"
```

Comment:
Use f-string for consistency

Reply:
Fixed in https://github.com/openvinotoolkit/datumaro/pull/1005/commits/161f31fccd9a5f7edbb853bb9f95d331d1709bfb